LinkBase/LinkBase/PeerReviewAllViewController.swift

```diff
@@ -12,36 +12,93 @@ class PeerReviewAllViewController: UIViewController, UITableViewDataSource, UITa
 
     @IBOutlet weak var tableView: UITableView!
     
+    var testCases: [ScoreCard] = []
+    var testCase1: ScoreCard!
+    var t1AudioFile: URL!
+    var testCase2: ScoreCard!
+    var t2AudioFile: URL!
+    
+    var questions: CommsQuestionList!
+
+    
+    @IBOutlet weak var countLabel: UILabel!
+    
+    
+    
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-
+        
+        questions = CommsQuestionList()
+        
+        // set up test cases
+        
+        testCase1 = ScoreCard(playerName: "Bart")
+        testCase1.commsQuestionsAnswered.insert(questions.commList[0], at: 0)
+        
+        // set up date - test case is based on dates about 1 week from current date
+        let currentDate = Date()
+        let tc1Date = currentDate - 500000
```

Comment:
we should probably make a method to say (one week ago) just to clean this up, but that can be later